projects/1.20/assets/mowzies-mobs/mowziesmobs/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+  "config.mowziesmobs.play_boss_music": "播放Boss音乐",
+  "config.mowziesmobs.custom_boss_bar": "自定义Boss生命槽",
+  "config.mowziesmobs.glow_effect": "显示荧光效果",
+  "config.mowziesmobs.custom_player_anims": "自定义玩家动作",
+  "config.mowziesmobs.crane_heal_sounds": "播放烈阳鸟愈鹭音效",
+
+  "_comment": "进度",
+  "advancements.mowziesmobs.root.title": "Mowzie的怪物",
+  "advancements.mowziesmobs.root.description": "欢迎来玩Mowzie的怪物!",
+  "advancements.mowziesmobs.kill_umvuthi.title": "赞美太阳",
+  "advancements.mowziesmobs.kill_umvuthi.description": "击杀乌穆武蒂, 太阳神鸟",
+  "advancements.mowziesmobs.kill_ferrous_wroughtnaut.title": "纵使巨大...",
+  "advancements.mowziesmobs.kill_ferrous_wroughtnaut.description": "击杀钢铁守护者",
+  "advancements.mowziesmobs.kill_foliaath.title": "珍尼弗甩了我",
+  "advancements.mowziesmobs.kill_foliaath.description": "击杀巨噬叶",
+  "advancements.mowziesmobs.kill_umvuthana_raptor.title": "假面舞会",
+  "advancements.mowziesmobs.kill_umvuthana_raptor.description": "击杀烈阳鸟袭禽",
+  "advancements.mowziesmobs.kill_frostmaw.title": "野蛮觉醒",
+  "advancements.mowziesmobs.kill_frostmaw.description": "击杀霜冻巨兽",
+  "advancements.mowziesmobs.kill_naga.title": "飞龙在天",
+  "advancements.mowziesmobs.kill_naga.description": "击杀娜迦",
+  "advancements.mowziesmobs.kill_grottol.title": "你简直...",
+  "advancements.mowziesmobs.kill_grottol.description": "击杀岩壳居蟹",
+  "advancements.mowziesmobs.sneak_grove.title": "一树之鸟",
+  "advancements.mowziesmobs.sneak_grove.description": "在伪装状态下潜入烈阳鸟巢穴",
+  "advancements.mowziesmobs.suns_blessing.title": "太阳法师",
+  "advancements.mowziesmobs.suns_blessing.description": "接受太阳祝福",
+  "advancements.mowziesmobs.steal_ice_crystal.title": "没头脑时间",
+  "advancements.mowziesmobs.steal_ice_crystal.description": "从霜冻巨兽处偷走冰霜结晶",
+  "advancements.mowziesmobs.kill_grottol_fortune.title": "搏一搏! ",
+  "advancements.mowziesmobs.kill_grottol_fortune.description": "使用附有时运魔咒的镐击杀岩壳",
+  "advancements.mowziesmobs.kill_grottol_silk_touch.title": "跟我走吧",
+  "advancements.mowziesmobs.kill_grottol_silk_touch.description": "使用附有精准采集魔咒的镐击杀岩壳",
+
+  "subtitles.mowziesmobs.foliaath.bite": "巨噬叶: 啃咬",
+  "subtitles.mowziesmobs.foliaath.pant": "巨噬叶: 喘息",
+  "subtitles.mowziesmobs.foliaath.grunt": "巨噬叶: 咕哝声",
+  "subtitles.mowziesmobs.foliaath.emerge": "巨噬叶: 出现",
+  "subtitles.mowziesmobs.foliaath.retreat": "巨噬叶: 缩回",
+  "subtitles.mowziesmobs.foliaath.die": "巨噬叶: 死亡",
+  "subtitles.mowziesmobs.foliaath.hurt": "巨噬叶: 受伤",
+  "subtitles.mowziesmobs.foliaath.rustle": "巨噬叶: 沙沙声",
+  "subtitles.mowziesmobs.foliaath.baby_eat": "巨噬叶幼苗: 吞吃",
+  "subtitles.mowziesmobs.wroughtnaut.whoosh": "千锻巨斧: 嗖嗖声",
+  "subtitles.mowziesmobs.wroughtnaut.creak": "钢铁守护者: 吱呀作响",
+  "subtitles.mowziesmobs.wroughtnaut.grunt": "钢铁守护者: 咕哝声",
+  "subtitles.mowziesmobs.wroughtnaut.hurt": "钢铁守护者: 受伤",
+  "subtitles.mowziesmobs.wroughtnaut.pre_swing": "钢铁守护者: 起手",
+  "subtitles.mowziesmobs.wroughtnaut.pull": "钢铁守护者: 拔斧子",
+  "subtitles.mowziesmobs.wroughtnaut.release": "钢铁守护者: 拔出斧子",

```

| **审阅级别** |
|:--------------:|
|❓ **疑问** / 不阻止合并|

原文：**Ferrous Wroughtnaut pulls his axe┃Ferrous Wroughtnaut frees his axe**
```suggestion
  "subtitles.mowziesmobs.wroughtnaut.pull": "钢铁守护者: 拔斧子",
  "subtitles.mowziesmobs.wroughtnaut.release": "钢铁守护者: 拔出斧子",
```
这俩啥区别

---

动画上，一个是正在努力拔斧子的音效，一个是把斧子拔出来的音效